checks/dangerous_workflow.go

```diff
@@ -179,6 +182,9 @@ func validateSecretsInPullRequests(workflow *actionlint.Workflow, path string,
 	if usesPullRequestTarget {
 		triggers[triggerPullRequestTarget] = usesPullRequestTarget
 	}
+    if usesWorkflowRun {
```

Comment:
1. bad indentation
2. did you use triggers[triggerPullRequestTarget] deliberately, or did you forget to edit after the copy-paste?

Reply:
oops, should've commented on the line that's below it